ui/src/submissions/institutions/containers/InstitutionSubmissionSuccessPageContainer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import SubmissionSuccess from '../../common/components/SubmissionSuccess';
+import { INSTITUTIONS } from '../../../common/routes';
+
+export const InstitutionSubmissionSuccessPage = ({ cnum, recordId }) => (
+  <SubmissionSuccess
+    message={
+      <span>
+        Successfully submitted, thank you for the submission! See the
+        institution ({cnum}){' '}
```

Comment:
cnum? this is institution submission right?

Reply:
maybe it's worth explaining what is this acronym cause it might not be so straightforward ;) https://inspire-schemas.readthedocs.io/en/latest/schemas/elements/cnum.html

Reply:
Deleted cnum